doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,488 @@
+.. _ncs_release_notes_160:
+
+Changes in |NCS| v1.6.0
+#######################
```

Comment:
Make sure to clean the changelog since the contents have been moved here. Also, the link to the changelog should be removed from doc/nrf/release_notes.rst
```suggestion
|NCS| v1.6.0 Release Notes
##########################
```